domains/florent.json

```
@@ -0,0 +1,18 @@
+{
+  "repo": "https://github.com/flobsx",
+  "owner": {
+    "username": "florent",
```

username doesn't match github username
```suggestion
    "username": "flobsx",
```